src/api/resources/classification/classification_status.py

```diff
@@ -0,0 +1,63 @@
+from django.db.models import OuterRef, Q, Subquery
+from django_filters import rest_framework as filters
+
+from ...exceptions import check_uuid
+from ...models import ClassificationStatus, CollectRecord, ObsBenthicPhotoQuadrat
+from ...permissions import ProjectDataAdminPermission, ProjectDataCollectorPermission
+from ..base import BaseAPIFilterSet, BaseAPISerializer, BaseProjectApiViewSet
+
+
+class ClassificationStatusSerializer(BaseAPISerializer):
+    class Meta:
+        model = ClassificationStatus
+        fields = ["id", "image", "status", "message", "data"]
+
+
+class ClassificationStatusFilterSet(BaseAPIFilterSet):
+    collect_record = filters.BooleanFilter(field_name="collect_record_id", method="filter_covars")
+
+    def filter_covars(self, queryset, name, value):
```

Comment:
The name `filter_covars` is confusing to me. Isn't it something more like `filter_crs`?